tests/Feature/Api/V1Test.php

```diff
@@ -0,0 +1,401 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use LegacyApp\Platform\Enums\AchievementType;
+use LegacyApp\Platform\Enums\UnlockMode;
+use LegacyApp\Platform\Models\Achievement;
+use LegacyApp\Platform\Models\Game;
+use LegacyApp\Platform\Models\PlayerAchievement;
+use LegacyApp\Platform\Models\System;
+use LegacyApp\Site\Models\StaticData;
+use LegacyApp\Site\Models\User;
+use Tests\TestCase;
+
+class V1Test extends TestCase
+{
+    use RefreshDatabase;
+
+    private User $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        /** @var User $user */
+        $user = User::factory()->create();
+        $this->user = $user;
+    }
+
+    private function apiUrl(string $method, array $params = []): string
+    {
+        $params = array_merge(['y' => $this->user->APIKey], $params);
```

Comment:
Don't the V1 URLs require `z=User` too? Isn't just relying on the token unsafe?

Reply:
The api token isn't much safer with the username - if it's known the username is, too.
There is no better practice to follow other than making sure the tokens are safe by themselves.
The endpoints already do not require the `z` parameter.
Api tokens will be replaced in the next iteration of the web api including personal token management in the settings area.